rust/Cargo.toml

```diff
@@ -17,51 +17,57 @@ rust-version = "1.76.0"
 # We are interested in testing subset combinations of this feature, so this is redundant
 denylist = ["unstable"]
 # To exercise combinations of any of these features, remove from `always_include_features`
-always_include_features = ["anyhow", "pyo3-asyncio-attributes", "dev-tools", "bumble-tools"]
+always_include_features = ["pyo3-asyncio-attributes", "dev-tools", "bumble-tools"]
 
 [dependencies]
-pyo3 = { version = "0.18.3", features = ["macros"] }
-pyo3-asyncio = { version = "0.18.0", features = ["tokio-runtime"] }
+pyo3 = { version = "0.20.2", features = ["macros", "anyhow"] }
+pyo3-asyncio = { version = "0.20.0", features = ["tokio-runtime"] }
 tokio = { version = "1.28.2", features = ["macros", "signal"] }
 nom = "7.1.3"
-strum = "0.25.0"
-strum_macros = "0.25.0"
+strum = "0.26.1"
+strum_macros = "0.26.1"
 hex = "0.4.3"
-itertools = "0.11.0"
+itertools = "0.12.1"
 lazy_static = "1.4.0"
 thiserror = "1.0.41"
 bytes = "1.5.0"
 pdl-derive = "0.2.0"
 pdl-runtime = "0.2.0"
 futures = "0.3.28"
+async-trait = "0.1.77"
+uuid = "1.8.0"
 
 # Dev tools
 file-header = { version = "0.1.2", optional = true }
 globset = { version = "0.4.13", optional = true }
 
 # CLI
-anyhow = { version = "1.0.71", optional = true }
```

Comment:
No longer optional due to wanting a "don't care" error for callbacks, etc, that wasn't the Python-specific `PyError`